### DISCUSSION TITLE
Don't  login

### DISCUSSION THREAD

**Discussion by Jengro777** [medusajs/medusa#5024] [Q&A]
I confirmed my account password correctly
----

![image](https://github.com/medusajs/medusa/assets/35259040/26603e52-45fa-44f7-b255-eb3edd040ea4)

------
![image](https://github.com/medusajs/medusa/assets/35259040/e7ad8d3e-7a52-466f-b98a-0540fe9534c1)

-------
![image](https://github.com/medusajs/medusa/assets/35259040/0e5010a8-1a7e-4d02-b439-0ea3e49efe02)

------
![image](https://github.com/medusajs/medusa/assets/35259040/98ae4e99-cebf-48ee-9320-1879a003d3e8)

------
```
webpack 5.88.2 compiled successfully in 2103 ms
::1 - - [12/Sep/2023:09:32:35 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:35 +0000] "GET /admin/store/ HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:35 +0000] "GET /admin/auth HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:36 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:36 +0000] "GET /admin/store/ HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:36 +0000] "GET /admin/auth HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:41 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:42 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:47 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:48 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
::1 - - [12/Sep/2023:09:32:59 +0000] "OPTIONS /admin/auth HTTP/1.1" 204 0 "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"

```

**Comment by pevey** [comment]
Check the applications tab to see if the cookie is being set.  Code 204 is the response you want here, so the backend is recognizing your password as correct.  All good there.  You can narrow your focus to why the browser is not setting the cookie.  It looks like you might be using Safari.  Note that Apple is more strict about cookies and will not set cookies cross domain.  In other words, your storefront, admin, and backend all need to be on the same root domain.  They can be on different subdomains.  They can be on different servers by pointing the subdomains to those servers via DNS.  

Example: 
storefront: mystore.com
backend: medusa.mystore.com
admin: medusa.mystore.com/app OR (if deployed separately) admin.mystore.com

**Comment by ayobamy** [comment]
> ## I confirmed my account password correctly
> ![image](https://private-user-images.githubusercontent.com/35259040/267276966-26603e52-45fa-44f7-b255-eb3edd040ea4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNTI0NDAsIm5iZiI6MTcyNzM1MjE0MCwicGF0aCI6Ii8zNTI1OTA0MC8yNjcyNzY5NjYtMjY2MDNlNTItNDVmYS00NGY3LWIyNTUtZWIzZWRkMDQwZWE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDEyMDIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI5ZjMyNjFiZjQ4YmM4MTgxNjdhZDcwODliMTE0ZDFkZDc0Zjc4ZDQ1ZWU1OGU1NWQ2ODBjMTcwNjQyNzk5ZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.6M7iKso1pykkgi8yRnTnwRsr8-BfmhlpxNrsBq1YkLo)
> 
> ![image](https://private-user-images.githubusercontent.com/35259040/267277399-e7ad8d3e-7a52-466f-b98a-0540fe9534c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNTI0NDAsIm5iZiI6MTcyNzM1MjE0MCwicGF0aCI6Ii8zNTI1OTA0MC8yNjcyNzczOTktZTdhZDhkM2UtN2E1Mi00NjZmLWI5OGEtMDU0MGZlOTUzNGMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDEyMDIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliOGViOWQxMjMzM2VmODc2ODQyMmZjMTAyOWZlNmI0N2NjN2MwMWM1Yjc0NTI2NjEwNDNlODVhMjljM2E5MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9H5GCOb24c3suH65H8KAnYJR96Zl7pmDXAMlHOAKYgA)
> 
> ![image](https://private-user-images.githubusercontent.com/35259040/267277940-0e5010a8-1a7e-4d02-b439-0ea3e49efe02.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNTI0NDAsIm5iZiI6MTcyNzM1MjE0MCwicGF0aCI6Ii8zNTI1OTA0MC8yNjcyNzc5NDAtMGU1MDEwYTgtMWE3ZS00ZDAyLWI0MzktMGVhM2U0OWVmZTAyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDEyMDIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkNDNiMzI5YzljZjBhZWNhYTYxZTk5YzFiMzRlMWY0ZDc4MWMzNDFhMTk0ZDVmN2E5YWI5ODIzODFkMzQ2YWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.DcuJCkNd7B4kxKxQQQSDTU7hCx4ZzO-Wn_y_9lyydgk)
> 
> ![image](https://private-user-images.githubusercontent.com/35259040/267278421-98ae4e99-cebf-48ee-9320-1879a003d3e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNTI0NDAsIm5iZiI6MTcyNzM1MjE0MCwicGF0aCI6Ii8zNTI1OTA0MC8yNjcyNzg0MjEtOThhZTRlOTktY2ViZi00OGVlLTkzMjAtMTg3OWEwMDNkM2U4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDEyMDIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0Njc1NzIxMmIxMmRiOGZlOGZlYzU4ODAxYTU3Njg2ZTA5MzhlYjNmYWZhOTkzYzQyMjY4MzNjNDFiNjMzYzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dwreYnmhWbKMbIEikqUMXE2kHj3k9dH6ejlRJCeRxvM)
> 
> ```
> webpack 5.88.2 compiled successfully in 2103 ms
> ::1 - - [12/Sep/2023:09:32:35 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:35 +0000] "GET /admin/store/ HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:35 +0000] "GET /admin/auth HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:36 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:36 +0000] "GET /admin/store/ HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:36 +0000] "GET /admin/auth HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:41 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:42 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:47 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:48 +0000] "GET /admin/batch-jobs?limit=100 HTTP/1.1" 401 - "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ::1 - - [12/Sep/2023:09:32:59 +0000] "OPTIONS /admin/auth HTTP/1.1" 204 0 "http://192.168.3.7:7001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 Edg/116.0.1938.76"
> ```

Have you resolved this issue yet.
I'm having a similar issue on here
